includes/class-sender.php

```diff
@@ -36,6 +37,41 @@ public static function init() {
 
 		// remote delete posts
 		add_action( 'webmention_delete', array( static::class, 'send_webmentions' ) );
+		\add_action( 'init', array( self::class, 'register_postmeta' ), 11 );
+	}
+
+	/**
+	 * Register post meta
+	 */
+	public static function register_postmeta() {
```

Comment:
```suggestion
	public static function register_meta() {
```

The other function is called `register_meta` I think we should be consistent. I have no preference which one to use.

Reply:
Same, moved it around...fine with changing it.